tests/network/cluster_addons_operator/test_network_addons_operator.py

```diff
@@ -214,6 +214,7 @@ def net_add_op_bridge_attached_vm(namespace, net_add_op_br1test_nad):
 
 
 @pytest.mark.gating
+@pytest.mark.core_setup
```

Comment:
This feedback is in the sig-network context.

I do not understand nor know what a `core setup` represents or means.
I find it challenging to maintain a group of tests by such a definition. We barely manage to maintain well the `gating` one, so why would we be interested in adding another one to this?

I think in the past a suggestion was raised to mark tests per a clear classification of what they cover or what they depend on (and not on notions like "special cluster").
The test runner, e.g. a Jenkins job, will select which tests to run.

Is there a proposal paper with a clear requirement and use case to have something like this?